docs/Tutorial.md

```diff
@@ -37,6 +67,17 @@ ONE-vscode has 6 screen components.
 
 <!-- Rearrange content below -->
 
+## Workspace
+
+You need to open a folder (`File > Open Folder` menu) to start with ONE-vscode. After opening a folder, a [_workspace_ is constructed by vscode.](https://code.visualstudio.com/docs/editor/workspaces).
+
+The folder for ONE-vscode has the following files:
+- Your model files
```

Comment:
`.circle` and profiling related files are excluded intentionally?

Reply:
I think model file (and maybe ONE file) is the most important for readers. Intermediate files are actually not a concern for them. So I didn't mentioned those files. 
(In gcc tutorial, maybe source code and a.out would be mentioned while there are many other intermediate files)
For json file, I will explain in Profiling section later. :-)